great_expectations/core/util.py

```diff
@@ -417,16 +417,16 @@ def convert_to_json_serializable(  # noqa: C901, PLR0911, PLR0912
     )
 
 
-def ensure_json_serializable(data):  # noqa: C901, PLR0911, PLR0912
+def ensure_json_serializable(data) -> None:  # noqa: C901, PLR0911, PLR0912
     """
     Helper function to convert an object to one that is json serializable
     Args:
         data: an object to attempt to convert a corresponding json-serializable object
-    Returns:
-        (dict) A converted test_object
     Warning:
         test_obj may also be converted in place.
     """
+    if isinstance(data, pydantic.BaseModel):
+        return
```

Comment:
This is the corresponding logic in `convert_to_json_serializable`

https://github.com/great-expectations/great_expectations/blob/a0b95a206feaaf12a0618f50f6c45caef7fecc1d/great_expectations/core/util.py#L250-L276